projects/igniteui-angular/src/lib/grids/columns/column.component.ts

```diff
@@ -1592,6 +1592,32 @@ export class IgxColumnComponent implements AfterContentInit, OnDestroy, ColumnTy
         return this._columnPipeArgs;
     }
 
+    /**
+     * @remarks
+     * Pass optional parameters to control properties of the default column editors.
+     * Accepts an `IColumnEditorOptions` object with the `dateTimeFormat` property.
+     * For more details see https://angular.io/api/common/DatePipe
```

Comment:
The DatePipe reference is no longer quite acurate now, right?

```suggestion
     * Pass optional properties for the default column editors.
     * @remarks
     * Options may be applicable only to specific column type editors. 
```
Honestly if there should be a link it'd be https://www.infragistics.com/products/ignite-ui-angular/angular/components/grid/column-types#datetime-date-and-time (which I suppose can use an update now too lol), but I'm not sure 
Also despite being the only option, the more specific description belongs to the `dateTimeFormat` option itself.